Exercise_2.java

```diff
@@ -1,3 +1,9 @@
+// Time Complexity : O(1)
+// Space Complexity : O(n)

```

Comment:
It is a good practice to define what 'n' is in the space/time complexity. 
Space Complexity : O(n) where n is the size of the stack